### ISSUE TITLE
Height of Events offset

### ISSUE THREAD

**Issue by Angeschossen** [aldabil21/react-scheduler#43]
Hello,

### Expected behavior:
The height of each event should be correct. There should be no offset at all.

### Observed behavior:
The height of the event gets a higher offset the lower it is.

![Image](https://i.imgur.com/ulAUVQV.png)

### Code
````
        return <div><div style={{margin: 20}}><Scheduler
                    remoteEvents={(q) => fetchRemote(q , this.props.videohub)}
                    onConfirm={(e, a) => {
                        return handleConfirm(e, a, this.props.videohub, this.props.output);
                    }}
                    onDelete={(id)=> handleDelete(id, this.props.videohub)}
                    view={"week"}
                    week={
                        {
                            weekDays: [0, 1, 2, 3, 4, 5, 6],
                            weekStartOn: 1,
                            startHour: 0,
                            endHour: 23,
                            step: 60,
                        }
                    }
                    fields={[
                        {
                            name: "title",
                            type: "select",
                            options: getOutputChoices(),
                            config: {
                                label: "Input", required: true, errMsg: "Please select an Input."
                            }
                        }
                    ]}
                    selectedDate={new Date()}
                    onEventDrop={(_date, updated, old) => {
                        updated.event_id = old.event_id;
                        return handleConfirm(updated, "edit", this.props.videohub, this.props.output);
                    }}
                /></div></div>;
````

Do you have any idea what the issue could be? Thanks in advance!

**Comment by aldabil21** [comment]
Hi @Angeschossen 

This heights issue is old before v2 which was fix here #9, I assume that you are not using a version before v2.

So i tried to put a sanbox that has the same setup as your code, but can't reproduce the height issue.

Can you continue from here: https://codesandbox.io/s/heights-test-y9ct17 and let me know if it works for you? If yes, then perhaps you have a style override that caused the issue, so try to copy your styles into the sanbox until the issue appear.

**Comment by Angeschossen** [comment]
> Hi @Angeschossen
> 
> The height issue is old before v2 which was fix here #9, I assume that you are not using a version before v2.
> 
> So i tried to put a sanbox that has the same setup as your code, but can't reproduce the height issue.
> 
> Can you continue from here: https://codesandbox.io/s/heights-test-y9ct17 and let me know if it works for you? If yes, then perhaps you have a style override that caused the issue, so try to copy your styles into the sanbox until the issue appear.

You're right. I'm sorry. Thanks a lot for your help! It was some global styling that the nextjs app setup command automatically added. My bad. The only issue left, is that the event do not fill the complete width of the column.

The image is taken of the examples linked at your repo. Is there any reason for this, or is it a mistake?
![img](https://imgur.com/qdSNjZR.png)

**Comment by aldabil21** [comment]
You're welcome.

> The only issue left, is that the events do not fill the complete width of the column, if there is no overlapping event.

This is intentional by design as shown here: https://github.com/aldabil21/react-scheduler/blob/c79e123d82154f10b5065ea45ee284414954311f/src/lib/styles/styles.ts#L105

The reason for that, so if you have an event, you would still have a little space to click on the grid/square below it to add another event at the same time.

Though, you could override it in your theme setup if you would like, and make it a 100%.
